### PR TITLE
22614 - Only allow 1 citizenship for Canadians.

### PR DESCRIPTION
**Pull request by hfekete**
*Issue:22614 *https://github.com/bcgov/entity/issues/22614 

*Description of changes:*
- convert citizenships from combobox to figma design (3 radios and 1 combobox)
- update tests 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-205-iq1kevhf.web.app

**Review by patrickpeinanw — APPROVED** [review]
Nice work! I just have some comments and questions

File: btr-web/btr-main-app/components/individual-person/add-new/index.vue [hunk 1]
```diff
@@ -281,23 +281,12 @@
         :border="editMode"
         no-top-border
       >
-        <div class="flex-col w-full">
-          <BcrosInputsCombobox
-            id="countriesOfCitizenship"
+        <div class="flex flex-col w-full">
+          <IndividualPersonCitizenship
```
[the remark this answers is not in the document]
Comment (hfekete) [reply btr-web/btr-main-app/components/individual-person/add-new/index.vue:285]: I am not sure about that.

**Pull request merged**